hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/transfer/CustomFeeAssessmentStep.java

```diff
@@ -215,13 +220,17 @@ private record CustomFeeAssessmentResult(
             List<CryptoTransferTransactionBody> assessedTxns, List<AssessedCustomFee> assessedCustomFees) {}
 
     private void validateTotalAdjustments(final CryptoTransferTransactionBody op, final int maxTransfersDepth) {
-        final var hbarTransfers = op.transfersOrElse(TransferList.DEFAULT)
-                .accountAmountsOrElse(emptyList())
+        final var hbarTransfers = op.transfersOrElse(TransferList.DEFAULT).accountAmountsOrElse(emptyList()).stream()
+                .map(AccountAmount::accountID)
+                .collect(Collectors.toSet())
```

Comment:
It shouldn't matter because this is a set, but will this code's implementation deterministically map to equivalent set objects every time? Or is this another subtle case where we might get an occasional ISS?

Reply:
It just needs size. So it doesn't matter if they are in order

Reply:
oh I missed that, nevermind thanks!